app/src/test/java/io/github/droidkaigi/confsched2017/viewmodel/TopicColorTest.kt

```diff
@@ -0,0 +1,44 @@
+package io.github.droidkaigi.confsched2017.viewmodel
+
+import io.github.droidkaigi.confsched2017.model.Topic
+import org.junit.Test
+
+import org.junit.Assert.*
+import org.assertj.core.api.Assertions.assertThat;
+
+class TopicColorTest {
+
+    @Test
+    fun from() {
+
+        // topic is null
+        run {
+            assertThat(TopicColor.from(null)).isEqualTo(TopicColor.NONE)
+        }
+
+        // invalid topic id
+        run {
+            assertThat(TopicColor.from(Topic().apply { id = -1 })).isEqualTo(TopicColor.NONE)
+            assertThat(TopicColor.from(Topic().apply { id = 10 })).isEqualTo(TopicColor.NONE)
+        }
+
+        // valid topic id
+        run {
+            assertThat(TopicColor.from(Topic().apply { id = TopicColor.PRODUCTIVITY_AND_TOOLING.ordinal }))
```

Comment:
I think provided `id` should be concrete value like this. `id = 1`, `id = 2`, `id = 3`.